src/Game.java

```diff
@@ -1,13 +1,64 @@
-import java.util.ArrayList;
+import java.util.Date;
+import java.util.Random;
+import java.util.Scanner;
 
 public class Game {
-  static ArrayList<Zombie> zombies = new ArrayList<Zombie>();
   static Board board = new Board(4, 4);
 
+  private static void printOptions() {
+    System.out.println("1: Place sunflower");
+    System.out.println("2: Place peashooter");
+    System.out.println("3: Skip Turn");
```

Comment:
Let's add a restart option in the future

Reply:
Agreed but i dont think we need it in this PR.